apiserver/remotefirewaller/ingressaddresswatcher_test.go

```diff
@@ -302,3 +311,62 @@ func (s *addressWatcherSuite) TestTwoUnitsSameAddressOneLeaves(c *gc.C) {
 	wc.AssertChange()
 	wc.AssertNoChange()
 }
+
+func (s *addressWatcherSuite) TestSeesMachineAddressChanges(c *gc.C) {
```

Comment:
We need a test where a second unit is assigned to an existing machine with a unit already on it